src/EntityFramework.SqlServer/SqlProviderServices.cs

```diff
@@ -132,6 +133,16 @@ public static bool TruncateDecimalsToScale
         }
 
         /// <summary>
+        /// Set this flag to false to prevent ordering using a row_number() function for queries with OFFSET operation, and order by columns directly instead.
```

Comment:
If we get good results in the beta I think we can revisit removing this flag altogether.